services/iombian-installed-services-downloader/0.1.0/docker-compose.yaml

```diff
@@ -0,0 +1,40 @@
+services:
+  iombian-installed-services-downloader:
+    image: ghcr.io/tknika/iombian-installed-services-downloader:0.1.0
+    container_name: iombian-installed-services-downloader
+    restart: unless-stopped
+    networks:
+      - iombian-internal-services
+    volumes:
+      - ${BASE_PATH}:${BASE_PATH}
+    environment:
+      CONFIG_HOST: "iombian-config-file-handler"
+      CONFIG_PORT: ${CONFIG_PORT}
+      BASE_PATH: ${BASE_PATH}
+      LOG_LEVEL: ${LOG_LEVEL}
+    labels:
+      com.iombian-installed-services-downloader.service.name: "IoMBian Installed Services Downloader"
+      com.iombian-installed-services-downloader.service.author: "Aitor Castaño"
+      com.iombian-installed-services-downloader.service.version: "0.1.0"
+      com.iombian-installed-services-downloader.service.description: "Download the services from firebase to sync the firebase installed services and the local installed services."
+      com.iombian-installed-services-downloader.service.changelog: "First release."
+      com.iombian-installed-services-downloader.service.documentation_url: ""
+
+      com.iombian-installed-services-downloader.env.CONFIG_PORT.name: "Config port"
+      com.iombian-installed-services-downloader.env.CONFIG_PORT.description: "The port of the IoMBian Config File Handler."
+      com.iombian-installed-services-downloader.env.CONFIG_PORT.type: "integer:1024:65535"
+      com.iombian-installed-services-downloader.env.CONFIG_PORT.default: "5555"
+
+      com.iombian-installed-services-downloader.env.BASE_PATH.name: "Base path"
+      com.iombian-installed-services-downloader.env.BASE_PATH.description: "The path where the iombian services will be added."
+      com.iombian-installed-services-downloader.env.BASE_PATH.type: "string"
+      com.iombian-installed-services-downloader.env.BASE_PATH.default: "/opt/iombian-services"
```

Comment:
Creo que es mejor opción no permitir que los usuarios puedan modificar el base path, porque puede romper muchas cosas y dejar el dispositivo inútil. Vamos a forzar a que el valor del volume sea: "/opt/iombian-services:/opt/iombian-services" 😉